backend/internal/common/env_config.go

```diff
@@ -76,4 +90,35 @@ func init() {
 	if parsedAppUrl.Path != "" {
 		log.Fatal("PUBLIC_APP_URL must not contain a path")
 	}
+
+	validTraceExporters := []string{"http", "grpc", "stdout"}
```

Comment:
For HTTP and GRPC, shouldn't we rely on the env var [`OTEL_EXPORTER_OTLP_PROTOCOL`](https://opentelemetry.io/docs/languages/sdk-configuration/otlp-exporter/#otel_exporter_otlp_protocol) (or the more specific `OTEL_EXPORTER_OTLP_TRACES_PROTOCOL` and `OTEL_EXPORTER_OTLP_METRICS_PROTOCOL`)?

So `METRICS_EXPORTERS` and `TRACES_EXPORTERS` would only used to enable stdout/prometheus, and all OTEL stuff would be configured via the `OTEL_*` env vars?

Reply:
Mmm, good point. I could use the `autoexport` package to make that simpler. I'll take a look at that.